staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/versionconverter_test.go

```diff
@@ -36,11 +36,17 @@ var testTypeConverter = func() TypeConverter {
 	if err != nil {
 		panic(err)
 	}
-	spec := spec.Swagger{}
-	if err := json.Unmarshal(data, &spec); err != nil {
+	swag := spec.Swagger{}
+	if err := json.Unmarshal(data, &swag); err != nil {
 		panic(err)
 	}
-	typeConverter, err := NewTypeConverter(&spec, false)
+
+	convertedDefs := map[string]*spec.Schema{}
```

Comment:
Can this call the conversion function?

Reply:
unfortunately not since it is in a different package.  This file is `package internal`. The helper function is in `package internal_test`. I might actually just remove the conversion function since I was only able to use it once.